core/src/main/kotlin/io/fritz2/dom/html/events.kt

```diff
@@ -1,21 +1,22 @@
 package io.fritz2.dom.html
 
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.FlowPreview
 import org.w3c.dom.*
 import org.w3c.dom.clipboard.ClipboardEvent
 import org.w3c.dom.events.*
 import org.w3c.xhr.ProgressEvent
 
-@ExperimentalCoroutinesApi
-@FlowPreview
-open class EventType<T>(val name: String) {
-    open fun extract(event: Event): T = event.unsafeCast<T>()
+/**
+ * [EventType] contains the javascript related [name] of an [Event]
+ * and has a method [extract] to cast an [Event] to its actual event-type.
+ */
+class EventType<T: Event>(val name: String) {
+    fun extract(event: Event): T = event.unsafeCast<T>()
```

Comment:
function call is unnecessary now. Just include it where it is called now `offer(....)`.